outport/interface.go

```diff
@@ -3,20 +3,20 @@ package outport
 import (
 	"github.com/ElrondNetwork/elrond-go/core/statistics"
 	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/indexer"
 	"github.com/ElrondNetwork/elrond-go/data/state"
-	"github.com/ElrondNetwork/elrond-go/outport/types"
 )
 
 // Driver is an interface for saving node specific data to other storage.
 // This could be an elastic search index, a MySql database or any other external services.
 type Driver interface {
-	SaveBlock(args types.ArgsSaveBlocks)
-	RevertBlock(header data.HeaderHandler, body data.BodyHandler)
-	SaveRoundsInfo(roundsInfos []types.RoundInfo)
+	SaveBlock(args *indexer.ArgsSaveBlockData)
```

Comment:
does this interface matches the new indexer implementation?

Reply:
Yes. 
This interface was thought for the new indexer implementation.

Reply:
great :+1: